src/api/parser/test/sanitize-html.test.js

```diff
@@ -0,0 +1,151 @@
+    const data = sanitizeHTML(
+      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https%3A%2F%2Fmedium.com%2F%40pedrofonsecadev%2Ftelescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
+    );
+    expect(data).toBe(
+      '<iframe src="https://open.spotify.com/embed/album/21wMUhXhWLew2zsWQhlYEM?referrer=https%3A%2F%2Fmedium.com%2F%40pedrofonsecadev%2Ftelescope-note-feb-22-2021-8601dfbaa1f0"></iframe>'
+    );
+  });
+
+  test('cdn.embedly.com embedded content should not get removed', () => {
+    const data = sanitizeHTML(
+      '<iframe src="https://cdn.embedly.com/widgets/media.html?src=https%3A%2F%2Fopen.spotify.com%2Fembed%2Falbum%2F21wMUhXhWLew2zsWQhlYEM&amp;display_name=Spotify&amp;url=https%3A%2F%2Fopen.spotify.com%2Falbum%2F21wMUhXhWLew2zsWQhlYEM&amp;image=https%3A%2F%2Fi.scdn.co%2Fimage%2Fab67616d00001e02142b21bd73f912e8dfd72ade&amp;key=a19fcc184b9711e1b4764040d3dc5c07&amp;type=text%2Fhtml&amp;schema=spotify"></iframe>'
+    );
+    expect(data).toBe(
+      '<iframe src="https://cdn.embedly.com/widgets/media.html?src=https%3A%2F%2Fopen.spotify.com%2Fembed%2Falbum%2F21wMUhXhWLew2zsWQhlYEM&amp;display_name=Spotify&amp;url=https%3A%2F%2Fopen.spotify.com%2Falbum%2F21wMUhXhWLew2zsWQhlYEM&amp;image=https%3A%2F%2Fi.scdn.co%2Fimage%2Fab67616d00001e02142b21bd73f912e8dfd72ade&amp;key=a19fcc184b9711e1b4764040d3dc5c07&amp;type=text%2Fhtml&amp;schema=spotify"></iframe>'
+    );
+  });
+
+  test('medium.com embedded content including Gist code should not get removed', () => {
+    const data = sanitizeHTML(
+      '<iframe src="https://medium.com/media/e19a2999dc5dcb71e7a5ad152fdb2a2a"></iframe>'
+    );
+    expect(data).toBe(
+      '<iframe src="https://medium.com/media/e19a2999dc5dcb71e7a5ad152fdb2a2a"></iframe>'
+    );
+  });
+
+  test('<pre> with inline style, sanitize strips inline style', () => {
+    const data = sanitizeHTML('<pre class="brush: plain; title: ; notranslate">Hello World</pre>');
+    expect(data).toBe('<pre>Hello World</pre>');
+  });
+
+  test('<table> with multiple tags and links should work', () => {
+    const data = sanitizeHTML(
+      '<table cellpadding="0" align="center" style="margin-left: auto; margin-right: auto; text-align: center;" cellspacing="0" class="tr-caption-container"><tbody><tr><td style="text-align: center;"><a style="margin-left: auto; margin-right: auto;" href="www.senecacollege.ca"><img src="https://1.bp.blogspot.com/11.JPG" height="640" border="0" width="592"></a></td></tr><tr><td style="text-align: center;" class="tr-caption">The Final Product</td></tr></tbody></table>'
+    );
+    expect(data).toBe(
+      '<table><tbody><tr><td><a href="www.senecacollege.ca"><img src="https://1.bp.blogspot.com/11.JPG" /></a></td></tr><tr><td>The Final Product</td></tr></tbody></table>'
+    );
+  });
+
+  test('twitch.tv embedded content should not be removed', () => {
+    const data = sanitizeHTML('<iframe src="https://www.twitch.tv/0pensrc"></iframe>');
```

Comment:
Embedded twitch content doesn't come from here, should be from player.twitch.tv
Thought I added the test for this here though, strange it's not here

Reply:
Thanks for pointing it out, maybe it got messed up when rebasing